data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,633 @@ mission "Remnant: Face to Maw 2B"
+	name "Return Plume to Caelian"
+	description "Plume wants to check his records of the research on void sprites to verify some information."
+	landing
+	source "Nasqueron"
+	destination "Aventine"
+	to offer
+		has "Remnant: Cognizance 6: done"
+	on offer
+		conversation
+			`After several hours Plume returns to the ship, spurred by the alarms in the suit warning him about his air levels. As you help him out of the suit he looks like he is having trouble deciding if he should be thrilled or worried.`
+			`	"It is wonderful to be here, watching them from a distance. A few did venture close, too. But I'm worried about them. Most of the void sprites out there have scorch marks similar to the scoring left by Korath weapons. And we already noticed that they are hiding much deeper in the atmosphere than normal." He glances down at his datapad then back up at you. "It looks like I will need to double-check my records and samples back on Aventine. Once we are back into the chaos of the Ember Waste I will send Chilia a signal."`
+				accept
+	npc
+		government "Korath"
+		personality staying
+		system "Nenia"
+		fleet
+			names "korath"
+			variant
+				"Korath Raider (Ember)" 3
+	on complete
+		conversation
+			`As you descend on Aventine, Plume gestures towards the pad closest to his lab. "You might as well set down there. If the samples in my lab check out, I will have another mission for you right away. You will need to keep your high-pressure ship active for this one. Stop by the lab in an hour or two and we will be ready to start loading."`
+
+
+
+mission "Remnant: Cognizance 8"
+	name "Transport engineer to Esquiline"
+	description "An engineer working for Plume needs transportation for himself and some equipment to Esquiline."
+	blocked "Plume's engineer needs a free bunk and 8 tons of cargo space for his equipment. Come back when you have enough space."
+	cargo equipment 8
+	passengers 1
+	source "Aventine"
+	destination "Esquiline"
+	to offer
+		has "Remnant: Cognizance 7: done"
+	on offer
+		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed esquiline station.`
+		conversation
+			`You walk into Plume's lab to be greeted by a large stack of crates and equipment. Plume and another remnant are gathered around the stack checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer, "This is Gavriil, one of our best technologists and a skilled astrobiologist in his own right. He will be going with you."`
+			`	"It will be a pleasure to work with you" sings Gavrill, making a short bow. "We will be traveling to a research station on Esquiline. Yes, it is a gas-world." He smiles at that. "I have to get some equipment setup and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`
```

Comment:
```suggestion
			`	"It will be a pleasure to work with you," sings Gavrill, making a short bow. "We will be traveling to a research station on Esquiline. Yes, it is a gas-world." He smiles at that. "I have to get some equipment setup and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`
```
Not sure "askance" is the word you're looking for here as it means "with suspicion or distrust."